themes/scss/map/overlays.scss

```diff
@@ -504,3 +504,10 @@ $index-items: 7 6 5 4 3 2 1;
 .Legend-placeholderInner {
   padding: 16px 24px 0;
 }
+
+
+@media (max-width: 759px) {
+  .CDB-Legends-canvas {
+    display: none !important;
```

Comment:
why don't you reverse the condition? something like this:
```
.CDB-Legends-canvas {
    display: none;
}

@media (min-width: 760px) {
    display: whatever;
}
```
it should wok without !important right?